packages/toolkit/utils/src/cli/prettyInstructions.ts

```diff
@@ -108,7 +108,7 @@ export const prettyInstructions = (appContext: any, config: any) => {
     const maxNameLength = Math.max(...routes.map(r => r.entryName.length));
 
     urls.forEach(({ label, url }) => {
-      message += `  ${chalk.bold(`> ${label}`)}\n`;
+      message += `  ${chalk.bold(`> ${label}`)}${chalk.cyanBright(url)}\n`;
```

Comment:
Maybe we should not print this URL in MPA, because the format look not good:

![image](https://github.com/web-infra-dev/modern.js/assets/7237365/99252f7d-ccce-437b-abdc-31d0ca627fe8)

Reply:
I missing this, should check `routes.length` ?

Reply:
it should be ok 👌